### PR TITLE
Added code for clear bits from msb in C, Java and Python

### PR DESCRIPTION
**Pull request by rbjoshi1309**
**Fixes issue:** #4780 

**Changes:**

Added C, Java and Python version for clear bits from msb in following folder :
cosmos/code/bit_manipulation/src/clear_bits_from_msb

### CODE REVIEW

File: code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Clear all MSB's except for the first i LSB's of a number n.
+MSB -> Most Significant Bit
+LSB -> Most Significant Bit 
+*/
+
+import java.util.*;
+
+class ClearBitsFromMsb
+{ 
+    static int clear_bits_from_msb(int n, int i)
```
Comment (arnavb) [inline code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.java]: Use camel case.

File: code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/*
+Problem Statement :
+Clear all MSB's except for the first i LSB's of a number n.
+MSB -> Most Significant Bit
+LSB -> Most Significant Bit
+*/
+
+#include <stdio.h>
+
+int clear_bits_from_msb(int n, int i)
```
Comment (arnavb) [inline code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.c]: Use camel case.

File: code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.java (hunk 1 above)
Comment (rbjoshi1309) [reply code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.java]: Done.

File: code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.c (hunk 2 above)
Comment (rbjoshi1309) [reply code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.c]: Done.

File: code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+'''
+Problem Statement :
+Clear all MSB's except for the first i LSB's of a number n.
+MSB -> Most Significant Bit
+LSB -> Most Significant Bit 
+'''
+
+def clearBitsFromMsb(n, i):
```
Comment (arnavb) [inline code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.py]: For Python keep lowercamel case. Also reformat this file using black.

Comment (rbjoshi1309) [reply code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.py]: Done.

**Pull request merged**